### ISSUE TITLE
Update js2coffee

### ISSUE THREAD

**Issue by alexandrosm** [jonschlinkert/grunt-js2coffee#1]
Any chance you can upgrade to version 0.2.2 as it has some features I would love to have?

Thanks

**Comment by jonschlinkert** [comment]
No prob, I would accept a pr as well

**Comment by alexandrosm** [comment]
Cool. Looking at the source I see you have js2coffee embedded as a lib instead of a node module. Any reason in particular, or should I do the PR simply moving it to node_modules?

**Comment by jonschlinkert** [comment]
Yeah, I made some changes to the source and fixed some bugs since it wasn't being maintained at the time. I'm not sure where it is now, but I'm happy to to move it into node_modules if that's useful to you

**Comment by alexandrosm** [comment]
It's a more general matter, really. If it was in node_modules, it would be trivial to update and keep updated. But if your changes are vital to the operation of the task, and they have not been reflected upstream, then we have a problem. So these are my questions now:
1. are the changes you made vital to the operation of the grunt task?
2. have you tried to get them merged upstream?
3. do the latest versions cover your changes?

thanks

**Comment by jonschlinkert** [comment]
I just took a quick look at the project and it's definitely better to use node_modules now. There was zero activity on the project before, but it appears to be in really good shape now.

**Comment by alexandrosm** [comment]
super. Will give it a shot as soon as I can and hopefully won't break
anything.

On Sat, Oct 19, 2013 at 5:14 AM, Jon Schlinkert notifications@github.comwrote:

> I just took a quick look at the project and it's definitely better to use
> node_modules now. There was zero activity on the project before, but it
> appears to be in really good shape now.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/jonschlinkert/grunt-js2coffee/issues/1#issuecomment-26643031
> .

## 

_Alexandros Marinos_

Founder / Director, Rulemotion Ltd

*
*

tel:****

+44 1483 68 8333

Surrey Technology Centre, 40 Occam Road****

mob:****

+44 7534 212 892**

Surrey Research Park****

twt:****

@alexandrosm https://twitter.com/#!/alexandrosm

Guildford GU2 7YG, UK****

---

Rulemotion Ltd is a limited company registered in England and Wales.
Company Registration No. 07794151. Registered Offices: 40 Occam Road,
Surrey Technology Centre, Surrey Research Park, Guildford, Surrey, GU2 7YG,
England.

IMPORTANT: This message from Rulemotion Ltd is private and confidential. If
you have received the message in error, please notify us and remove it from
your system.

**Comment by jonschlinkert** [comment]
:+1:
